packages/worker/src/entities/token.entity.ts

```diff
@@ -7,7 +7,7 @@ import { hexTransformer } from "../transformers/hex.transformer";
 import { BaseEntity } from "./base.entity";
 
 export enum TokenType {
-  ETH = "ETH",
+  ChainNative = "CHAINNATIVE",
```

Comment:
Please, use `BASE` here. And let's name it everywhere as a Base token.

Reply:
I've replaced every occurrence of native to Base.